style.css

```diff
@@ -7,6 +7,36 @@
   box-sizing: border-box;
 }
 
+@media (max-width: 1800px) {
```

Comment:
what happens on 1801px sceens?
maybe you meant "all above 1150px?"
then we should use `@media (min-width: 1151px)`

Reply:
/+ the import file is not a place for styles, nor for reset (above you have `* { box-sizing: border-box; }` which is a part of css reset ), please move them to specific files